website/src/pages/chat/[id].tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Flex } from "@chakra-ui/react";
+import { List } from "lucide-react";
+import { GetServerSideProps } from "next";
+import Head from "next/head";
+import Link from "next/link";
+import { useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { Flags } from "react-feature-flags";
+import { ChatConversation } from "src/components/Chat/ChatConversation";
+import { getDashboardLayout } from "src/components/Layout";
+
+const Chat = ({ id }: { id: string }) => {
+  const { t } = useTranslation(["common", "chat"]);
+
+  return (
+    <>
+      <Head>
+        <title>{t("chat")}</title>
+      </Head>
+
+      <Flags authorizedFlags={["chat"]}>
+        <Flex direction="column" gap="2">
+          <Link href="/chat">
+            <Button leftIcon={<List />} size="lg">
+              {t("chat:back_to_chat_list")}
+            </Button>
+          </Link>
+
+          <ChatConversation chatId={id as string} />
```

Comment:
Isn't the id already string?

Reply:
true, this is a remanent from and old idea I had where the id was part of the query parameters, and in that case, `id` was of type `string | string[]`, I will change it after work.